src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/DependencyInjection/DashboardExtension.php

```diff
@@ -59,6 +59,19 @@ public function load(array $configs, ContainerBuilder $container): void
         }
 
         $this->parseTeamsConfiguration($config['teams'], $container);
+
+        $container->setParameter(
```

Comment:
As we like to debundle, why not configuring this in `parameters:` instead?

Reply:
This was later removed. :wastebasket: